portal-api/src/drivers/fileshare.js

```diff
@@ -165,9 +165,11 @@ const deleteMultipleFiles = async (fileshare, filePath, fileList) => {
   if (!fileList) return false;
 
   if (Array.isArray(fileList)) {
-    return fileList.map(async (filename) => {
-      await deleteFile(fileshare, `${filePath}/${filename}`);
-    });
+    return Promise.all(
```

Comment:
```suggestion
    const promises = await Promise.all( ...

    return promises;
```

Reply:
Could I possibly ask what the reason for this change is? Also with the note that, semantically, the thing that gets returned when a `Promise.all` is `await`ed isn't a list of `promises`, but instead a list of whatever the `Promise` resolved to

Reply:
I've found this pattern to be helpful when debugging, including returning what is resolved in the promise(s)

Reply:
Because this function is marked as `async`, it will always return a list of resolved promises as long as the function is called with `await` (ie. `return Promise.all` is the same as `return await Promise.all` in terms of the result)